index.html

```diff
@@ -180,6 +192,8 @@ <h3>CPU registers</h3>
             <img src="imgs/loading.gif" id="loading_img" alt="loading..." />
             <select name="romchoice" id="romchoice">
                 <option value="">Loading...</option>
+                <!-- Actually, you can add default prebuild here. Here is an example: -->
```

Comment:
Pre-built images are fetched from a remote index, no need to hardocde the values here

Reply:
Deleted.